.prettierrc.js

```diff
@@ -1,7 +1,4 @@
 module.exports = {
-    jsxBracketSameLine: true,
     trailingComma: 'all',
-    tabWidth: 4,
-    arrowParens: 'avoid',
-    // printWidth: 120,
+    printWidth: 120,
```

Comment:
I remember Lubo specifically requesting this and I agree that the default 80 chars feels a bit restricted on todays monitors.

Reply:
It is roughly the same as what is recommended for Realm Core.

Reply:
This is great.  Excited for 2 spaces and a constrained width.  120 seems appropriate even for working directly on the laptop monitor